docs/content/developers/building-contributing.md

```diff
@@ -1,12 +1,13 @@
 # Building, Testing, and Contributing
 
-## Testing Latest Commits
+## Testing Latest Commits on HEAD
 
-You can download the latest artifacts from the master branch from [link](https://nightly.link/drud/ddev/workflows/master-build/master). On macOS and Linux, you can `brew unlink ddev && brew install drud/ddev/ddev --HEAD --fetch-HEAD` to get the latest commit of DDEV, even if it’s unreleased. If you’re using Linux/WSL2, you’ll likely need to install build-essential with the following command: `sudo apt install -y build-essential`.
+You have many options for getting and using the latest committed HEAD version of DDEV:
 
-To download the latest version, you can visit the [master-build](https://github.com/drud/ddev/actions/workflows/master-build.yml) workflow and choose the latest item (or the one that matches a commit you want to test). The artifacts for each OS are at the bottom of the page.
-
-Alternately, you can see the latest build in action by visiting DDEV on [Gitpod](https://gitpod.io/#https://github.com/drud/ddev).
+* **Download** the latest artifacts from the master branch from [link](https://nightly.link/drud/ddev/workflows/master-build/master). This version is signed and notarized, built by the CI system. Get the one you need and place it in your `$PATH`.
```

Comment:
“link” isn’t descriptive text, but “nightly.link” at least hints at where they’ll live. Reworded for brevity. Used “each of these” to transition from plural (“artifacts”) to singular items instead of jumping straight to “This version”. Reordered for chronological accuracy (I think).

```suggestion
* **Download** the latest master branch artifacts from [nightly.link](https://nightly.link/drud/ddev/workflows/master-build/master). Each of these is built by the CI system, signed, and notarized. Get the one you need and place it in your `$PATH`.
```